src/test/java/hw8/JdiSiteTest.java

```diff
@@ -0,0 +1,46 @@
+package hw8;
+
+import com.epam.jdi.light.driver.WebDriverUtils;
+import com.epam.jdi.light.elements.init.PageFactory;
+import hw8.data.MetalsColorsFormDataProvider;
+import hw8.entities.JdiUser;
+import hw8.entities.MenuElements;
+import hw8.entities.MetalsColorsInput;
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import static java.lang.String.format;
+
+public class JdiSiteTest {
+
+    @BeforeSuite(alwaysRun = true)
+    public void beforeSuite() {
+        PageFactory.initSite(JdiSite.class);
+    }
+
+    @AfterSuite(alwaysRun = true)
+    public void afterSuite() {
+        WebDriverUtils.killAllSeleniumDrivers();
+    }
+
+    @Test(priority = 1)
```

Comment:
What is the reason for using priority here? Is this possible to create just one single test?

Reply:
Of course it is possible to create one single test. But I don't find it reasonable:
Login method is a kind of precondition for the Metals & Colors Page test, but as we have assertions in there I haven't marked it with "BeforeMethod" or "BeforeTest" annotation.
Also there are multiple data sets for the Metals & Colors Page test. If we unify Login and Metals & Colors in one test, Login wil be repeted each run. What's the point if we can just refresh the Metals & Colors page?
That's why I used priority. But it's also possible to use "dependsOnMethods", and maybe it's even more reasonable.

Reply:
If you insist, I'll unite tests in one